lib/utils/logDeprecation.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+const chalk = require('chalk');
+const memoizee = require('memoizee');
```

Comment:
It'll be cleaner to use `memoizee/weak`, then memoization will no lock service objects in a memory

Reply:
Done